src/components/AdminSettings/WebServerSetupChecks.vue

```diff
@@ -0,0 +1,177 @@
+import JitsiStreamBackgroundEffect from '../../utils/media/effects/virtual-background/JitsiStreamBackgroundEffect'
+import VirtualBackground from '../../utils/media/pipeline/VirtualBackground'
+
+export default {
+	name: 'WebServerSetupChecks',
+
+	directives: {
+		tooltip: Tooltip,
+	},
+
+	data() {
+		return {
+			backgroundBlurLoaded: undefined,
+		}
+	},
+
+	computed: {
+		backgroundBlurAvailable() {
+			return this.backgroundBlurLoaded
+		},
+
+		backgroundBlurAvailableClasses() {
+			return {
+				'icon-checkmark': this.backgroundBlurAvailable === true,
+				'icon-error': this.backgroundBlurAvailable === false,
+				'icon-loading-small': this.backgroundBlurAvailable === undefined,
+			}
+		},
+
+		backgroundBlurAvailableAriaLabel() {
+			if (this.backgroundBlurAvailable === false) {
+				return t('spreed', 'Failed')
+			}
+
+			if (this.backgroundBlurAvailable === true) {
+				return t('spreed', 'OK')
+			}
+
+			return t('spreed', 'Checking …')
+		},
+
+		backgroundBlurAvailableToolTip() {
+			if (this.backgroundBlurAvailable === false && !VirtualBackground.isWasmSupported()) {
+				return t('spreed', 'Failed: WebAssembly is disabled or not supported in this browser. Please enable WebAssembly or use a browser with support for it to do the check.')
+			}
+
+			if (this.backgroundBlurAvailable === false) {
+				return t('spreed', 'Failed: ".wasm" and ".tflite" files were not properly returned by the web server. Please check "System requirements" section in Talk documentation.')
+			}
+
+			if (this.backgroundBlurAvailable === true) {
+				return t('spreed', 'OK: ".wasm" and ".tflite" files were properly returned by the web server')
+			}
+
+			return t('spreed', 'Checking …')
+		},
+	},
+
+	beforeMount() {
+		this.checkBackgroundBlur()
+	},
+
+	methods: {
+		checkBackgroundBlur() {
+			if (!VirtualBackground.isWasmSupported()) {
+				this.backgroundBlurLoaded = false
+
+				return
+			}
+
+			this.backgroundBlurLoaded = undefined
+
+			// Pass only the essential options to check if the files can be
+			// loaded.
+			const options = {
+				virtualBackground: {
+					type: VIRTUAL_BACKGROUND_TYPE.NONE,
+				},
+				simd: VirtualBackground.isWasmSimd(),
+			}
+
+			/* eslint-disable no-undef, camelcase */
+
+			// When the worker is loaded from Talk its URL starts with
+			// "apps/spreed/js". However, when it is loaded from the
+			// administration settings its URL starts with "apps/talk/js"
+			// instead, so it fails to load.
+			//
+			// "publicPath" option in "worker-loader" configuration does not
+			// work with Webpack 5. As a workaround the public path needs to be
+			// overriden at runtime before loading the worker and restored
+			// afterwards.
+			// https://github.com/webpack-contrib/worker-loader/issues/281
+			const __webpack_public_path__saved = __webpack_public_path__
+
+			__webpack_public_path__ = __webpack_public_path__.replace('talk', 'spreed')
```

Comment:
I got this in my console:

> GET https://nextcloud24.local/apps/spreed/js/talk-JitsiStreamBackgroundEffect.worker.worker.js?v=1472aec776b99b3ad271
[HTTP/1.1 404 Not Found 0ms]
> 10:54:01.876 Laden des Workers von "https://nextcloud24.local/apps/spreed/js/talk-JitsiStreamBackgroundEffect.worker.worker.js?v=1472aec776b99b3ad271" wurde auf Grund eines nicht freigegebenen MIME-Typs ("text/html") blockiert.

And a never finishing loading spinner.

For me the path should be `/appspriority/spreed/js/`

The following code (with importing the function from nextcloud/router package) works:
```suggestion
			__webpack_public_path__ = generateFilePath('spreed', 'js', '')
```